poet-tslint.json

```diff
@@ -22,20 +21,27 @@
       "no-unused-variable": true,
```

Comment:
Do we have a chore for this: `no-unused-variable is deprecated. Since TypeScript 2.9. Please use the built-in compiler checks instead.`?

Reply:
yes, removing prettier is a pretty big task so I was holding off on that for another PR.